qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/ClusteringRule.java

```diff
@@ -511,6 +532,7 @@ public void stopBroker(final int nodeId) {
           broker.getConfig().getNetwork().getCommandApi().getAddress();
       final List<Integer> brokersLeadingPartitions = getBrokersLeadingPartitions(socketAddress);
       broker.close();
+      closeables.remove(broker);
```

Comment:
nice catch

Reply:
So close was also called twice? But this should be cleared after the test class?

Ah ok and if you run a test in a loop this piles up right

Reply:
Where do you see that the brokers are added to the closeables? Haven't found this in the code?

Reply:
Uh, you're right, they aren't. I added it because I ran into this issue in the past and had to do that, apparently it was already removed, my bad :sweat_smile: Makes more sense to not have the brokers managed by closeables anyway